packages/flutter/lib/src/widgets/automatic_keep_alive.dart

```diff
@@ -89,6 +89,10 @@ class _AutomaticKeepAliveState extends State<AutomaticKeepAlive> {
         // build of this subtree. Wait until the end of the frame to update
         // the child when the child is guaranteed to be present.
         SchedulerBinding.instance.addPostFrameCallback((Duration timeStamp) {
+          // The element is no longer alive
```

Comment:
This seems reasonable. When writing the code we were actually wondering if that case could ever happen and thought it wouldn't....

Can you add a test for this case?

Reply:
Done